server/main_server/src/irods_server_control_plane.cpp

```diff
@@ -367,20 +367,20 @@ namespace irods
         rodsEnv my_env;
         _reloadRodsEnv( my_env );
 
-        int re_pid = 0;
+        int delay_server_pid = 0;
         // no error case, resource servers have no re server
         try {
-            re_pid = get_server_property<int>(irods::RE_PID_KW);
+            delay_server_pid = get_server_property<int>(irods::RE_PID_KW);
```

Comment:
RE_PID_KW... is this... available to be changed, or due to being a KW... it's tougher to change?

Reply:
I'll remove it as part of the delay server migration PR since it won't be needed anymore.

Reply:
the best.

Reply:
See https://github.com/irods/irods/pull/6272#issue-1181913057